node_modules/@okta/i18n/package.json

```diff
@@ -1,16 +1,8 @@
 {
   "_args": [
     [
-      {
-        "raw": "@okta/i18n@19.76.0",
-        "scope": "@okta",
-        "escapedName": "@okta%2fi18n",
-        "name": "@okta/i18n",
-        "rawSpec": "19.76.0",
-        "spec": "19.76.0",
-        "type": "version"
-      },
-      "/Users/lboyette/okta/okta-signin-widget"
+      "@okta/i18n@19.76.0",
+      "/Users/stephen.tsai/okta/okta-signin-widget"
```

Comment:
which version of npm do you have? just wondering why the structure changed here.